### DISCUSSION TITLE
Location issue on New meeting

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#1475] [General]
On our current site I am trying to add a new meeting that is online only. The did want to include the city so I clicked no for attending in person and just used the drop down for the city. It saved as yes you can attend in-person. I selected no again, removed the location saved and same thing, it switches to the yes option. I cannot this meeting to not show in-person in the listing.

**Comment by joshreisner** [comment]
> just used the drop down for the city

every meeting should have a location entered in the "Address" field, even if it's just a city, eg `Durham, NC`. this is independent of whatever is selected in the Region dropdown menu

**Comment by district6wc** [comment]
Hi Josh the problem is I do that and it flips to in-person and this meeting is online only and I cannot get the radio button to stick it flips to yes for I can attend in-person on save.

**Reply by joshreisner** [reply]
could you show a screenshot of how the screen looks before you hit save?

**Reply by joshreisner** [reply]
for example here is a meeting i created on my test site - i gave it an approximate location, and time, and day, and was able to publish with `No` for "Can I Attend In Person?" - is this similar to how you have it?

![Screenshot 2024-07-26 at 14-20-18 Edit Meeting “Test Meeting” ‹ WordPress Test — WordPress](https://github.com/user-attachments/assets/c423ef35-b584-4685-8ac5-288e5f575ce4)
![Screenshot 2024-07-26 at 14-20-36 Test Meeting – WordPress Test](https://github.com/user-attachments/assets/e77b8daa-475e-4b0f-a0cb-779d257b4c37)

**Comment by district6wc** [comment]
Josh here is a screen cast of what's happening, the no button switches to yes on save
https://somup.com/cZiO3VGofr

**Reply by joshreisner** [reply]
thank you, that's great. like i said above, every meeting should have a location entered in the "Address" field, even if it's just a city, eg Durham, NC. 

in your screencast the address field is empty

**Comment by district6wc** [comment]
Okay got it that worked thanksThank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328
